src/rdo.rs

```diff
@@ -405,8 +488,14 @@ pub fn compute_rd_cost<T: Pixel>(
   let bias = (mean_importance / FACTOR) as f64 + ADDEND;
   debug_assert!(bias.is_finite());
 
+  bias
+}
+
+pub fn compute_rd_cost<T: Pixel>(
+  fi: &FrameInvariants<T>, rate: u32, distortion: u64,
+) -> f64 {
```

Comment:
💯 Overall, separating the `compute_distortion_bias` from `compute_rd_cost` appears to simplify the RDO functions. I think the implicit contract is that distortion functions need return the biased distortion.